CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
```

Comment:
Are we all agreed to use this file to keep record of updates in each versions? I know there were conversations for using Changelog file. Just wanted to make sure :)

Reply:
@joncarlson Any thoughts on manual changelogs, automated changelogs, checklists, etc (from our other PR conversation)?

Reply:
My one reservation about changelogs is that historically they have not been kept up to date, which is why we haven't implemented them in mEditor.

So the question is, if we start adding changelogs, how do we ensure they stay updated? Having a checklist, or even better, a merge check, would go along way to making sure the changelog is actually useful

Reply:
Agreed that a checklist / merge check seems like a really good way to remind people to update tests and documentation. I'd fully support that for all our repos, so I'll make a note to discuss this on Thursday.